db/seeds.rb

```diff
@@ -23,6 +23,7 @@
 
 require 'ffaker'
 require 'ruby-progressbar'
+include ActiveSupport::Testing::TimeHelpers
```

Comment:
I feel like we added this in #1567, I guess it will come up when you squash / rebase?